test/cusparse/linalg.jl

```diff
@@ -30,5 +30,9 @@ end
     @test Array(triu(A, 1)) ≈ triu(a, 1)
     @test Array(tril(A)) ≈ tril(a)
     @test Array(tril(A, 1)) ≈ tril(a, 1)
-    @test Array(exp(A)) ≈ exp(collect(a))
+    if CUSPARSE.version() > v"11.4.1"
+        @test Array(exp(A)) ≈ exp(collect(a))
+    else
+        @test_throws ErrorException Array(exp(A)) ≈ exp(collect(a))
```

Comment:
Why does the test fail before 11.4.1? We generally also don't use `@test_throws ErrorException` in that case, but just don't test the code path. On 1.7+ we could to `@test ... broken=CUSPARSE.version()<v"11.4"`, but we still support Julia 1.6 for now.

Reply:
The code fails because it seems that sparse-sparse multiplication is supported only on newer versions.